vimeo-networking/src/main/java/com/vimeo/networking/Configuration.java

```diff
@@ -101,27 +101,27 @@ public Cache getCache() {
      * Persist the {@link VimeoAccount} in the {@link AccountStore}
      * <p>
      * Account name is <b>required</b> for authenticated user accounts; for
-     * client credential account saving use {@link #saveAccount(VimeoAccount)}
+     * client credential account saving use {@link #saveNonUserAccount(VimeoAccount)}
      *
      * @param account     the account to persist
      * @param accountName the name of the account.
      */
-    public void saveAccount(@NotNull VimeoAccount account, @NotNull String accountName) {
+    public void saveAuthenticatedUserAccount(@NotNull VimeoAccount account, @NotNull String accountName) {
         if (accountStore != null) {
```

Comment:
I think the issue with changing the member variable names is the spider web effect - changing the name here will have implications on another class, which in turn may need name changes, and it can continue to branch out from there. 
How we handle this is something we should touch on in the next Android sync - do we ticket every model and their app - dependencies individually? Do we tackle each class when working on tickets related to the class?
I am wary of touching more variable renaming because of this spider web effect. There is a ticket to work on the Configuration/Config Builder already (https://vimean.atlassian.net/browse/VA-1473) so if anything, I would think that ticket would be more appropriate than this one to do that clean up.